_pages/team.html

```diff
@@ -43,7 +43,7 @@ <h5>Jack Evans</h5>
             <img src="/images/team/andrew-lester.png">
             <p class="label">🖥️ Programmer 🖥️</p>
             <h5>Andrew Lester</h5>
-            <p>Andrew is a programmer with years of experience in Java, Python, and HTML/CSS. He will handle our foundational robot code during the 2019 season and has worked on tasks from updating our website to maintaining our open-source projects.</p>
+            <p>Andrew is a programmer with years of experience in Java, Python, and HTML/CSS. He will handled our foundational robot code during the 2019 season and has worked on tasks from updating our website to maintaining our open-source projects.</p>
```

Comment:
Why?

Reply:
I think this is supposed to be: "he handled"